pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
This seems to be caused by missing "BIT64" def. I was adding that def in #126, [CMakeLists.txt](https://github.com/Microsoft/ChakraCore/pull/126/files#diff-af3b638bc2a3e6c650974192a53c7291). Could you please see if that change solves this issue?